### DISCUSSION TITLE
Bluetooth problem

### DISCUSSION THREAD

**Discussion by benischatz** [NightscoutFoundation/xDrip#3403] [Q&A]
Hello. Changed phones today. Downloaded latest xdrip 10.3.24 (Xiaomi f21 pro) transmitter connected but only gave 3 readings and will not reconnect. No battery optimization, play protect is off. Oop did not help..  it is working perfectly on a different phone.
Any ideas?? 
TIA

**Comment by Navid200** [comment]
Please have a look at this page:
https://navid200.github.io/xDrip/docs/Connectivity-troubleshoot.html

After you do, if the problem persists, please post a screenshot of the status page.

**Reply by Navid200** [reply]
That is assuming you are using a Dexcom device.  If not, please ignore all of that and clarify what you are using.

**Reply by Navid200** [reply]
When you are editing your post, you can click on the paper clip symbol to attach an image.

**Reply by benischatz** [reply]
It's taking time to upload

**Reply by Navid200** [reply]
Edit the image file and scale it down to make it a smaller file.  As long as it can be seen, there is no need to have extremely high resolution.

**Reply by benischatz** [reply]
![Uploading IMG_20240326_205320.jpg…]()

**Reply by benischatz** [reply]
![Screenshot_2024-03-27-02-18-43-859_com miui home](https://github.com/NightscoutFoundation/xDrip/assets/97632459/8b5ef7ab-d28d-4f3a-93ec-6bb0474548b5)
![Uploading Screenshot_2024-03-27-02-17-11-910_com.whatsapp.jpg…]()

**Reply by Navid200** [reply]
OK,

So, you have changed phones today and this I suppose is the new phone.

Have you disabled collection on the old phone?
https://navid200.github.io/xDrip/docs/Stop-xDrip.html

**Reply by benischatz** [reply]
![Uploading Screenshot_2024-03-27-02-17-11-910_com.whatsapp.jpg…]()

**Reply by benischatz** [reply]
Sure. Been trying to figure this out for a while with tzachi...

**Reply by benischatz** [reply]
Restart collector only works for one reading

**Reply by Navid200** [reply]
Please answer this question:
Have you disabled collection on the old phone?

**Reply by benischatz** [reply]
Sure

**Reply by benischatz** [reply]
Disabled transmitter from old Bluetooth and turned phone off

**Reply by Navid200** [reply]
Sorry, that makes no sense, disabling transmitter from old Bluetooth.
Are you sure the old phone is off?  To avoid having to ask this question, do you mind disabling collection on the old phone?
Here again how to do it: https://navid200.github.io/xDrip/docs/Stop-xDrip.html

**Reply by benischatz** [reply]
Ok. Trying again

**Reply by Navid200** [reply]
Thanks.

Also, this line should never have been translated in xDrip.
![317080414-8b5ef7ab-d28d-4f3a-93ec-6bb0474548b5](https://github.com/NightscoutFoundation/xDrip/assets/51497406/038cafb0-a312-4ae2-8cb3-ccdcdc076879)

I have opened a submission so that it is not translated:
https://github.com/NightscoutFoundation/xDrip/pull/3303
Hopefully, it will be reviewed.
But, that's not your problem.  It is mine.

Would you please tell me what that line translates to?

**Reply by benischatz** [reply]
It says: two minutes ago
I disabled on both phones. Old one turned off. New one not finding 
![Screenshot_20240327-024447](https://github.com/NightscoutFoundation/xDrip/assets/97632459/3cb71f39-3256-42b3-b5af-9dd436a20b70)

**Reply by benischatz** [reply]
Sorry. Got first reading now. Awaiting next one

**Reply by benischatz** [reply]
We have four readings so far!! Don't know what's changed as i have done all the above before, but it is 3 Am by us and i am happy enough as it is now..
Would like to thank you so very much for being so patient with me!!!
I happen to give a lot of xdrip support on WhatsApp groups but i guess there is always more to learn . So thanks again!

**Reply by Navid200** [reply]
You're welcome

**Reply by benischatz** [reply]
![Screenshot_20240327-080350](https://github.com/NightscoutFoundation/xDrip/assets/97632459/a85c8e8c-8cae-4f2b-9547-2b66a0ed168c)

**Reply by benischatz** [reply]
![Screenshot_20240327-080517](https://github.com/NightscoutFoundation/xDrip/assets/97632459/03142b63-357a-4a56-9a30-8ddf7f4a4915)

**Reply by benischatz** [reply]
I'm back :(

**Reply by Navid200** [reply]
You cannot have more than one phone attempt to connect to the same transmitter at the same time.  Have you disabled collection on the old phone?

Are you sure you have gone over all of these?
https://dontkillmyapp.com/xiaomi

Do you have a Dexcom receiver to use as a test to verify that the transmitter is not malfunctioning?
If not, do you have a phone that is compatible with the official Dexcom app for the same reason?

Do you have another phone for a test that is neither Xiaomi nor Samsung?

**Reply by benischatz** [reply]
So this is the story: i never have 2 phones on. I always disable collection, turn one off and then the other on. 
I've been through all battery optimizations. 
Our second phone is a jelly 2, it works perfectly!! So transmitter is ok!!!
I've downgraded now to older version (24.10.23) it connected once and no more readings...
Any other ideas??

**Reply by Navid200** [reply]
Do I understand correctly that you never have this disconnect problem with the jelly 2 phone?

**Reply by benischatz** [reply]
Correct. Only the Xiaomi is problematic.
It used to work fine, then had to be remanufactured and problems are ever since

**Reply by benischatz** [reply]
![Screenshot_20240327-205757](https://github.com/NightscoutFoundation/xDrip/assets/97632459/d077b6eb-6d53-437c-a1c4-7e50fd3d9cd7)

**Reply by benischatz** [reply]
But Bluetooth is on...

**Reply by Navid200** [reply]
Xiaomi is a known problem phone.  

You can try enabling or disabling avoid scanning or minimize scanning to see if it makes any difference.

**Reply by benischatz** [reply]
It helped for an hour only :(
